catroid/src/main/java/org/catrobat/catroid/ui/ProgramMenuActivity.java

```diff
@@ -82,7 +82,7 @@ public void onCreate(Bundle savedInstanceState) {
 			Sprite sprite = ProjectManager.getInstance().getCurrentSprite();
 			Scene scene = ProjectManager.getInstance().getCurrentScene();
 			if (sprite != null && scene != null && actionBar != null) {
-				String title = scene.getName() + ": " + sprite.getName();
+				String title = ProjectManager.getInstance().getCurrentProject().getSceneList().size() > 1 ? scene.getName() + ": " + sprite.getName() : sprite.getName();
```

Comment:
All your comments and questions are welcome.
Different issues in software localization to BiDi-language should be considered, my progress is very  good, and  I have faced many correlated challenging aspects for localization to these languages. The characteristics of BiDi-languages can raise issues in translation. many thanks.